README.md

```diff
@@ -33,7 +33,8 @@ The source for the m65dbg app is available here:
 # Building
 
 * You will need to install a few pre-requisite libraries (via apt-get/yum/zypper, or cygwin's setup.exe)
-  * **libreadline-devel**
+  * **libreadline-dev**
```

Comment:
Ah ok, the -dev / -devel thing seems unique to which linux flavour the package is for, but no worries, let's make it -dev, I think that's what the Ubuntu/debian packages prefer.

Reply:
Yes this was tested on lubuntu, so likely for ubuntu flavours. RPM is -devel. I could add explanation for all.